doc/release/platforms/README.sgi.rst

```diff
@@ -49,8 +55,7 @@ This can be done using the following settings:
 
```

Comment:
I think the purpose of keeping them alive is in case people are interested in these platforms, these readmes might be of more help than nothing. More importantly I think it serves as a good place for us to let them know to contact us if they do care about these systems.

I really don't think we want to update any of the actual content for ibm, marenostrum, or altix. And that's not just because it'd be more work for me :)

Would it help to make the warning more aggressive/forward and outright say that we know some of this information is wrong?